src/FSharpPlus/Operators.fs

```diff
@@ -97,7 +97,11 @@ module Operators =
     let inline (<*>) (f: '``Applicative<'T -> 'U>``) (x: '``Applicative<'T>``) : '``Applicative<'U>`` = Apply.Invoke f x : '``Applicative<'U>``
 
     /// Apply 2 lifted arguments to a non-lifted function.
-    let inline liftA2 (f: 'T->'U->'V) (x: '``Applicative<'T>``) (y: '``Applicative<'U>``) : '``Applicative<'V>`` = (f <!> x : '``Applicative<'U->'V>``) <*> y
```

Comment:
If you simply remove `liftA2` it would be a breaking change.

Could you deprecate it instead? As we did with other functions, like `valueOr` in Validation.fs

Reply:
It was moved to line 104 so we could use inside a `lift2` function without duplicating the logic inside.

Reply:
Indeed, sorry, my bad.